src/statsig/index.ts

```diff
@@ -148,9 +146,9 @@ export function getDefaultStatsigUser(): StatsigUser {
   const { store } = require('src/redux/store')
   const state = store.getState()
   return {
-    userID: walletAddressSelector(state) ?? undefined,
+    userID: state.web3.account?.toLowerCase() ?? undefined,
     custom: {
-      startOnboardingTime: startOnboardingTimeSelector(state),
+      startOnboardingTime: state.account.startOnboardingTime,
```

Comment:
removing these selectors allowed me to break the cyclical dependency of statsig -> walletAddressSelector in src/web3/selectors -> statsig. this cycle was causing the imports from statsig to be undefined, which caused tests to fail (since you cannot pass undefined to `createSelector`). i don't see any problems with having no userID / the previous userID during demo mode